pulsar-broker-common/src/main/java/org/apache/pulsar/common/configuration/PulsarConfigurationLoader.java

```diff
@@ -173,7 +173,8 @@ private static boolean isEmpty(Object obj) {
      *             if conf has the field whose name is not contained in ServiceConfiguration and ignoreNonExistMember is false.
      * @throws RuntimeException
      */
-    public static ServiceConfiguration convertFrom(PulsarConfiguration conf, boolean ignoreNonExistMember) throws RuntimeException {
+    public static ServiceConfiguration convertFrom(PulsarConfiguration conf, boolean ignoreNonExistMember,
```

Comment:
I would recommend not modifying these methods for the use case in the worker.  It doesn't really make sense to modify the method to allow override of one specific config. We should just do the override in the worker code itself.